WTWSMS/common/landed_titles/WtWSMS_pagan_religious_heads.txt

```diff
@@ -608,6 +608,31 @@ d_exilarch = {
 	dynasty_title_names = no 	# Will not be named "Seljuk", etc.
 }
 
+d_exilarch_karaite = {
+	color = { 155 216 230 }
+	color2 = { 220 220 0 }
+	
+	capital = 693 # Baghdad
+
+	title = "EXILARCH"
+	foa = "KOHEN_GADOL_FOA"
+	short_name = yes
+	
+	# Parent Religion 
+	religion = karaite
+	
+	culture = mizrahi
```

Comment:
Here too, I think you added the reference to the Mizrahi culture by accident.

Reply:
I have changed it to Aramaic

Reply:
I have changed it to Aramaic